content/docs/guides/contribute/code.en.md

```diff
@@ -17,7 +17,7 @@ Most OSRD developers use Linux (incl. [WSL](https://learn.microsoft.com/en-us/wi
 - Open a terminal[^git-bash] in the folder where the source code of OSRD will be located
 - Run `git clone git@github.com:osrd-project/osrd`
 
-### Launch the application with docker-compose
+### Launch the application with docker compose
```

Comment:
It's fine, although I would just link to the readme of OSRD once (and hopefully forever):
```suggestion
### Launching the application

Follow instructions from [OSRD's Readme](https://github.com/osrd-project/osrd#getting-started).
This will require `docker` and `docker compose`. [^package-manager][^docker-desktop].
```
(and clean almost all the rest of the section)